cmd/gcp-routes-controller/run.go

```diff
@@ -82,15 +97,14 @@ func runRunCmd(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create httpCheck: %v", err)
 	}
-
 	errCh := make(chan error)
 	tracker := &healthTracker{
 		state:            unknownTrackerState,
 		ErrCh:            errCh,
 		SuccessThreshold: 2,
-		FailureThreshold: 10,
-		OnFailure:        func() error { return exec.Command("systemctl", "stop", runOpts.gcpRoutesService).Run() },
-		OnSuccess:        func() error { return exec.Command("systemctl", "start", runOpts.gcpRoutesService).Run() },
+		FailureThreshold: 5, // it takes 20 seconds for vip changes to propagate, so give us 25 seconds before we react
```

Comment:
would even go down to 3

Reply:
yeah, fair enough; it's just kubelet, sdn, and multus that use this.